plugins/newsql/newsql.h

```diff
@@ -28,6 +28,14 @@ struct newsql_stmt {
     char tzname[CDB2_MAX_TZNAME];
 };
 
+typedef enum {
+    NEWSQL_STATE_NONE,
```

Comment:
nitpick: maybe open cdb2api could be updated to set `newsqlheader.state` to `NEWSQL_STATE_NONE` in `send_reset()`. It should be fine without this though because `state` will be default initialized to 0 and `NEWSQL_STATE_NONE` implicitly has the value 0!